### ISSUE TITLE
Problem with the format of the. log.json file output during the training process

### ISSUE THREAD

**Issue by W-QY** [open-mmlab/mmdetection#10173]
I would like to know where the content of the. json file printed by mmdetection is configured? Currently, there are issues with the. log. json file generated after using mmdet training (as shown in Figure 1). The first issue is that when I use the training mode of IterBasedRunner (steps=100000 runner=dict (type='IterBasedRunner ', max_iters=steps)), epoch always displays as 1. Another issue is the validation output during the training process, where the output iter is the number of images in the validation set. Figure 2 shows the correct output I saw on the network. I would like to know how to generate the. log. json file in the form shown in Figure 2.

![2{MA X7L7VW$S_NX(XHBW$9](https://user-images.githubusercontent.com/80522236/232731608-7345b90a-f7d6-47c7-a907-17ad1ea0d04b.png)

![OYML6M47ZY7GX50T5X(`1M2](https://user-images.githubusercontent.com/80522236/232731656-dce9a0be-8a06-44e5-ab8a-d29bd5e28d97.png)

**Comment by W-QY** [comment]
I would like to ask if you can give me a help. This error prevents me from observing the val results of the mmdet model in real time on wandb......................

**Comment by charlie10brown** [comment]
+1, I am also trying to acquire the validation loss. Anyone can help?
